model.go

```diff
@@ -7,27 +7,24 @@ package tflitego
 #cgo linux LDFLAGS: -lm -ldl -lrt
 */
 import "C"
-import (
-	"fmt"
-)
 
-// TFLiteModel represents a TensorFlow Lite Model.
-type TFLiteModel struct {
+// Model represents a TensorFlow Lite Model.
+type Model struct {
```

Comment:
I think is more clear use Model instead to TFLiteModel. I started with TFLiteModel to maintain compatibility with C library. 
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/c/c_api.h 

But seems more idiomatic for Go "Model"

Reply:
If we decide to change the package name, we'll have something like:

`tflite.Model`, and that's pretty verbose, I guess.